src/utils/ViewUtils.js

```diff
@@ -427,6 +428,15 @@ define(function (require, exports, module) {
             var fp = file.fullPath.split("/");
             fp.pop(); // Remove the filename itself
             displayPaths[index] = fp.pop();
+
```

Comment:
See if we can use useIconProviders and useClassProviders from Working Set View for this as well

Reply:
ping @swmitra  for guidance. I believe the "Remote" word can be added to the :after of a class and then that class can be registered with working set view.